.github/workflows/test.yml

```diff
@@ -44,9 +44,17 @@ jobs:
           else
               # test_assemblers.py is run below
               TEST_CATEGORY="not test_stages.py and not test_assemblers.py"
+
+              # DNF python package can't be installed using pip in the tox environment.
+              # We need to use the version from the system to test things.
+              # Since we are running tests on Fedora, enable site packages only
+              # for Python version which is available on Fedora.
+              if [ "${{ matrix.environment }}" == "py312" ]; then
```

Comment:
Perhaps a bit iffy to keep up to date with our `osbuild-ci` container, could we somehow use a test that sees if libdnf is available on the host instead?

Reply:
I agree that this bit will need to be manually adjusted every time the `osbuild-ci` is updated to a version that uses newer Python. And that it is not great.

Would you mind providing a bit more details on how would you envision this to work? The test is running in the tox environment and can't use libraries from the host, unless these are explicitly allowed. The problem here is that libdnf python bindings can't be installed using pip. And the host will always have libdnf installed for a specific python version, but not for all that we need to test. :thinking:

Reply:
We could maybe have a marker (file, or environment) in the `osbuild-ci` container that says for which version libdnf is available and use that in the shell script to determine when to include system packages?

Reply:
Thanks for the additional info. I'll look into it :+1:

Reply:
I've created https://github.com/osbuild/containers/pull/79